tests/test_uflash.py

```diff
@@ -322,11 +357,27 @@ def test_main_two_args():
         assert mock_flash.called_once_with('foo.py', '/media/foo/bar')
 
 
-def test_main_extra_args_ignored():
```

Comment:
I've removed this test because contorting the argparse parser to cope with it would be awkward, and I don't think it really made sense anyway. Why is it a good idea to just ignore command line args? Presumably the user expects them to mean something.